.gitignore

```diff
@@ -10,6 +10,9 @@
 /setup.h
 /*tags
 /autom4te.cache
+/.vs
```

Comment:
I'm fighting a losing battle, but I really think that all such standard directories should be ignored in the global `.gitignore` instead of doing it in each and every project-specific file.

Reply:
Not in every project the `.vs` dir should be ignored, so a global config can't be applied I think.

VS creates lots of files there, and until I ignore them, VS usually fails to perform Git operations like switching to a branch or a commit:

![image](https://github.com/wxWidgets/wxWidgets/assets/37658952/9cfbba79-02aa-4eb2-aec2-d91e5c54f0a3) 

![image](https://github.com/wxWidgets/wxWidgets/assets/37658952/950bc83c-564d-4e51-a9f7-c298fcfb5c4c)

Reply:
> Not in every project the .vs dir should be ignored, so a global config can't be applied I think.

Why not? I have it in mine since a very long time and didn't notice any problems due to this.

Reply:
Some projects require commiting specific files inside these dot directories.
And it would be more manual work for a new developer.